src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -35,7 +35,9 @@ import { FinderScope, TreeView } from './tree/TreeView'
 
 library.add(faTimes, faSearch)
 
-const DEFAULT_VISIBLE_TYPES = [EntityType.PROJECT, EntityType.FOLDER]
+const DEFAULT_SELECTABLE_TYPES = Object.values(EntityType)
+const TABLE_DEFAULT_VISIBLE_TYPES = Object.values(EntityType)
+const TREE_DEFAULT_VISIBLE_TYPES = [EntityType.PROJECT, EntityType.FOLDER]
```

Comment:
Using constants will prevent unnecessary rerenders when the default values are used.

Reply:
Nice catch